app/models/item_type.rb

```diff
@@ -4,4 +4,32 @@ class ItemType < ActiveRecord::Base
   validates :base_fee, :fee_per_day, numericality: { greater_than_or_equal_to: 0 }
 
   has_many :item
+
+  def cost(start_time, end_time)
+    return 0 if start_time > end_time
+
+    # have to add one day at the end 12th to 13th is 2 days, pick up on 12 drop of on 13 is two full days
+    rental_duration = (end_time.to_date - start_time.to_date).to_i + 1
+    no_of_weeks = (rental_duration / 7)
+    # Do not charge for 1/7 days in a rental.
+    days_to_charge_for = rental_duration - no_of_weeks
+    # Long term pricing
+    longterm_prices = longterm_cost(no_of_weeks)
+
+    unless longterm_prices.nil? # catch cases where item is neither 4 nor 6 seat cart
+      return longterm_prices[no_of_weeks] + ((rental_duration % 7) * fee_per_day)
+    end
+    (days_to_charge_for * fee_per_day) + base_fee
+  end
+
+  def longterm_cost(weeks)
```

Comment:
This seems like a *terrible* place to encode cost information. If there are more kinds of carts, you have to change your code. If you ever want to make changes to your costs (which you will), you have to change your code.

Ideally the people who care about money would be able to change the costs themselves, I imagine.

Reply:
It was just a copy paste from `app/models/rentals.rb`

Reply:
I mean, okay. Still seems like a bad idea.

Reply:
It is, I agree with you and I have created a separate issue to deal with this #361